application/ui/widgets/eh-widgets-dir/eh-widgets/src/page/bundle-group/BundleGroupPage.js

```diff
@@ -121,14 +132,28 @@ const BundleGroupPage = () => {
                           alt="BundleGroup Logo" />}
                     </div>
 
-                    {(pageModel.bundleGroup.displayContactUrl) &&
-                      <>
-                        <div className="BundleGroupPage-contact-us">
-                          <p>{i18n.t('page.bundleGroupInfo.contactUsInfo')}</p>
-                          <Button href={pageModel.bundleGroup.contactUrl} target="_blank">{i18n.t('component.button.contactUs')}</Button>
-                        </div>
-                        <hr/>
-                      </>
+                    {(pageModel.bundleGroup.displayContactUrl) && (pageModel.bundleGroup.contactUrl)
+                        && checkContactUsUrl(pageModel.bundleGroup.contactUrl) == false
```

Comment:
Getting eslint warnings here on startup, please switch to ===

Reply:
Updated to ===